src/lib/common/globals.h

```diff
@@ -116,7 +116,7 @@
 #define OPT_UNIQUE_VALUES   "unique"
```

Comment:
This PR merges to develop. The CNR in develop is still full of stuff from the las release …
I'll empty it and add my line.